lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -41,6 +43,19 @@ function restrictPropCoreFunctions()
             end
         end
     end
+
+    for _, signature in pairs( adminOnlyFunctions ) do
+        if wire_expression2_funcs then
+            local oldFunc = wire_expression2_funcs[signature][3]
+               wire_expression2_funcs[signature][3] = function( self, ... )
+               if ( self.player:IsAdmin() ) then
+                    return oldFunc( self, ... )
+               else
+                    self.player:ChatPrint( "You don't have access to " .. signature )
+               end
+            end
+        end
+    end
 end
 
```

Comment:
looks like theres some indentation issues that need to be solved
why is it indented after `local oldfunc = ...`
and not indented inside the function